File: day036/homework/code/codewars7.py
# https://www.codewars.com/kata/57aa218e72292d98d500240f/train/python

import math
def heron(a, b, c):
    eight=(a+b+c)/2
    კკ=eight*(eight-a)*(eight-b)*(eight-c)
    return (math.sqrt(კკ))

File: day068/homework/code/codewars7.js
// https://www.codewars.com/kata/515de9ae9dcfc28eb6000001/train/python

def solution(s):
    if len(s) % 2 != 0:
        s += '_'
    return [s[i:i+2] for i in range(0, len(s), 2)]
